sdk/src/main/scala/com/horizen/account/storage/AccountStateMetadataStorageView.scala

```diff
@@ -224,6 +214,18 @@ class AccountStateMetadataStorageView(storage: Storage) extends AccountStateMeta
         WithdrawalEpochCertificateSerializer.toBytes(certificate)))
     })
 
+    withdrawalEpochInfoOpt.foreach(epochInfo => {
```

Comment:
Probably `blockFeeInfoOpt.foreach ...`
and inside
` epochInfo = withdrawalEpochInfoOpt.getO(<get from storage>)`

Reply:
Done